dept44-api-models/pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>se.sundsvall.dept44</groupId>
+        <artifactId>dept44-parent</artifactId>
+        <version>3.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>dept44-api-models</artifactId>
```

Comment:
Maybe rename to `dept44-models`? 
It's a more generic name and not restricted to pojos för the api-layer.